policyengine_us/tests/policy/baseline/contrib/scott_budow/end_entrenched_poverty.yaml

```diff
@@ -0,0 +1,19 @@
+- name: One child single parent under poverty line
+  period: 2022
+  reforms: policyengine_us.reforms.scott_budow.end_entrenched_poverty.end_entrenched_poverty.end_entrenched_poverty_credit
+  input:
+    gov.contrib.scott_budow.end_entrenched_poverty.child[0].amount: 300
+    people:
+      head: {}
+      child: {}
+    tax_units:
+      tax_unit: 
+        members: [head, child]
+        tax_unit_children: 1
+        tax_unit_fpg: 1_000
+    households:
+      household:
+        members: [head, child]
+        household_net_income: 0
+  output:
+    end_entrenched_poverty_credit: 300
```

Comment:
```suggestion
    end_entrenched_poverty_credit: 300
    ny_refundable_credits: 300
```

Reply:
for this to work we might need to set all of those to 0

-    ny_eitc
    - ny_supplemental_eitc
    - ny_ctc
    - ny_cdcc
    - ny_real_property_tax_credit
    - ny_college_tuition_credit

Reply:
double check if test is written monthly